android/src/com/google/zxing/client/android/Intents.java

```diff
@@ -32,13 +32,16 @@ private Intents() {
    * Constants related to the {@link Scan#ACTION} Intent.
    */
   public static final class Scan {
-      
+
     public static final String USE_LED = "USE_LED";
     public static final String USE_FRONT_CAMERA = "USE_FRONT_CAMERA";
     public static final String SHOW_RECTANGLE = "SHOW_RECTANGLE";
     public static final String KEEP_OPEN = "KEEP_OPEN";
+    public static final String ALLOW_MENU = "ALLOW_MENU";
+    public static final String ALLOW_INSTRUCTIONS = "ALLOW_INSTRUCTIONS";
+    public static final String SHOW_INFOTEXT = "SHOW_INFOTEXT";
```

Comment:
Should be `SHOW_INFO_TEXT ` I think also `showInfoText` instead of `showInfotext`